tests/suites/test_suite_ecp.data

```diff
@@ -285,6 +305,10 @@ ECP read binary #16 (Curve448, non-canonical)
 depends_on:MBEDTLS_ECP_DP_CURVE448_ENABLED
 ecp_read_binary:MBEDTLS_ECP_DP_CURVE448:"ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff":"ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff":"0":"1":0
 
+ECP read binary #17 (non-zero, compressed format, p != 3 mod 4)
+depends_on:MBEDTLS_ECP_DP_SECP224R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP224R1:"0200000000000000000000000000000000000000000000000000000000":"01":"01":"01":MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE
```

Comment:
I think we should have this for the other SW curve with p != 3 mod 4, and we also should have a positive case for each curve which is supposed to work. (See note above regarding raising our testing standards.)

Reply:
I added an additional test case for secp224k1.

https://github.com/Mbed-TLS/mbedtls/issues/5940 already exists for generating code for all curves, and should do so for both compressed and uncompressed.  I really think requiring me to implement #5940 here is excessive scope creep for this PR.  If #5490 were merely two mins of work, then it would not exist as a separate work item.

Reply:
Good point, the correct strategy is indeed to have those cases generated, and adding a generator here would be scope creep. Marking this as resolved then.